logserver/logserver.c

```diff
@@ -950,40 +944,44 @@ int pv_logserver_send_vlog(bool is_platform, char *platform, char *src,
 		.src = src,
 	};
 
-	if (logserver.pid <= 0 || level > pv_config_get_log_loglevel())
+	if (level > pv_config_get_log_loglevel())
 		return -1;
```

Comment:
shouldn't this return 0? Doesn't look like an error if the level of the message doesn't make it because of loglevel

Reply:
Yes, you're right!